app/android/build.gradle

```diff
@@ -0,0 +1,31 @@
+buildscript {
+    ext.kotlin_version = '1.3.50'
```

Comment:
Good to note these version requirements somewhere, does the Flutter project handle it own dependencies or does the developer need to have these exact versions installed?

Speaking of, have you @MummenRider been keeping instructions on how to start the project from GitHub at all?

- [ ] Make a wiki page with instructions for fresh set up of project after pulling from GitHub (MacOS and windows)

Reply:
Yes, Flutter handle its own dependencies. 
I will create soon. As of now, I clone this specific branch on my windows and getting errors. Once I fixed these issues, I'll create instruction guide